lms/djangoapps/edxnotes/decorators.py

```diff
@@ -9,6 +9,7 @@
 from xblock.exceptions import NoSuchServiceError
```

Comment:
@alexjmpb why is that the id is passed with branch or version?

Reply:
That must be reviewed further, it was caused due a change of how the function is obtaining the course id. But there is more investigation to do, which will completed when the PR to upstream is created.